code/modules/research/xenobiology/crossbreeding/_clothing.dm

```diff
@@ -0,0 +1,131 @@
+/*
+Slimecrossing Armor
+	Armor added by the slimecrossing system.
+	Collected here for clarity.
+*/
+
+//Rebreather mask - Chilling Blue
+/obj/item/clothing/mask/nobreath
+	name = "rebreather mask"
+	desc = "A transparent mask, resembling a conventional breath mask, but made of bluish slime. Seems to lack any air supply tube, though."
+	icon_state = "slime"
+	item_state = "slime"
+	body_parts_covered = 0
+	w_class = WEIGHT_CLASS_SMALL
+	gas_transfer_coefficient = 0
+	permeability_coefficient = 0.5
+	flags_cover = MASKCOVERSMOUTH
+	resistance_flags = NONE
+
+/obj/item/clothing/mask/nobreath/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_WEAR_MASK)
+		user.add_trait(TRAIT_NOBREATH, "breathmask_[REF(src)]")
+		user.failed_last_breath = FALSE
+		user.clear_alert("not_enough_oxy")
+		user.apply_status_effect(/datum/status_effect/rebreathing)
+
+/obj/item/clothing/mask/nobreath/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_trait(TRAIT_NOBREATH, "breathmask_[REF(src)]")
+	user.remove_status_effect(/datum/status_effect/rebreathing)
+
+/obj/item/clothing/glasses/prism_glasses
+	name = "prism glasses"
+	desc = "The lenses seem to glow slightly, and reflect light into dazzling colors."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "prismglasses"
+	var/colour = "#FFFFFF"
+	actions_types = list(/datum/action/item_action/change_prism_colour, /datum/action/item_action/place_light_prism)
+
+/obj/item/clothing/glasses/prism_glasses/item_action_slot_check(slot)
+	if(slot == SLOT_GLASSES)
+		return TRUE
+
+/obj/structure/light_prism
+	name = "light prism"
+	desc = "A shining crystal of semi-solid light. Looks fragile."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "lightprism"
+	density = FALSE
+	anchored = TRUE
+	max_integrity = 10
+
+/obj/structure/light_prism/Initialize(var/colour)
+	. = ..()
+	color = colour
```

Comment:
this makes me very sad. just name the var glasses_color or something else that isn't so damn silly.